types/redux-first-router/index.d.ts

```diff
@@ -152,9 +183,15 @@ export interface Options {
     onAfterChange?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     onBackNext?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     restoreScroll?(history: History): ScrollBehavior;
+    initialDispatch?: boolean;
+    querySerializer?: QuerySerializer;
     navigators?: NavigatorsConfig;
 }
 
+// Params and Payload differ from Flow definition, as they can be arbitrarily shaped
+export type Params = any;
+export type Payload = any;
+
```

Comment:
no worries at all - thanks for the quick responses and the guidance, that's _exactly_ what I was looking for and didn't know quite how to express (`any` definitely ain't it). I'll get the (corrected) stuff in by tonight and circle back around